starfish/imagestack/imagestack.py

```diff
@@ -290,6 +292,43 @@ def xarray(self) -> xr.DataArray:
         """Retrieves the image data as an xarray.DataArray"""
         return self._data
 
+    def sel(self, indexers: Mapping[Indices, Union[int, tuple]]=None):
+        """Given a dictionary mapping the index name to either a value or a slice range, return an
+        Imagestack with each dimension indexed accordingly
+
+        Parameters
+        ----------
+        indexers : Dict[Indices, (int/tuple)]
+            A dictionary of dim:index where index is the value or range to index the dimension
+
+        Examples
+        --------
+        Imagestack stack with shape (5, 5, 15, 200, 200)
```

Comment:
I'd love these to be real code examples (see e.g. https://github.com/spacetx/starfish/blob/master/starfish/intensity_table/intensity_table.py#L42)